app/(main)/(screens)/collaboration-details/[pageID].tsx

```diff
@@ -0,0 +1,62 @@
+import CollaborationPage from "@/components/collaboration/CollaborationDetails";
+import { useEffect, useState } from "react";
+import { doc, getDoc } from "firebase/firestore";
+import { useLocalSearchParams } from "expo-router";
+import { FirestoreDB } from "@/shared-libs/utilities/firestore";
+import AppLayout from "@/layouts/app-layout";
+import { ActivityIndicator } from "react-native-paper";
+import { ICollaboration } from "@/shared-libs/firestore/trendly-pro/models/collaborations";
+
+interface Collaboration extends ICollaboration {
```

Comment:
Types can be shifted to types folder - file-name.ts, if these types are related to data not to component.